pyfolio/plotting.py

```diff
@@ -896,7 +896,8 @@ def plot_rolling_beta(returns, factor_returns, legend_loc='best',
     return ax
 
 
-def plot_rolling_volatility(returns, rolling_window=APPROX_BDAYS_PER_MONTH * 6,
+def plot_rolling_volatility(returns, factor_returns=pd.DataFrame(),
```

Comment:
I compared with plot_rolling_beta one definition above, and there, the factor_returns are always required (despite the docstring saying that it is optional).
But yeah, I can fix this.